Modification de `ENGINE` en `InnoDB` de la table `interactQuery` en v…

lucguinchard opened the pull request:
Pour tester cette PR, Il faut deux installations : 
- une fresh
- une migration

Puis, vérifier avec la requête SQL suivante que la table est bien en InnoDB : `ENGINE=InnoDB` (sur la dernière ligne de la réponse) : 
```SQL
show create table interactQuery;
```

lucguinchard committed:
warning fix lors de l'execution de cronDaily
PHP Warning:  Illegal string offset 'class' in /usr/share/nextdom/src/Helpers/NextDomHelper.php on line 859

lucguinchard committed:
Initialisation de l'ajout des widgets dans le core :
- Ajout du de l'entrée `widget` dans `nexdom.config.php`.
- Ajout des fichiers widgets : widgets.php Ajaxwidgets.php widgets.js widgets.scss ....widgetsManager.php.
- Ajout des templates spécifiques pour les widges.
- modification des version de Jeedom et Nextdom.
- Ajout dans le menu un bouton `widgets`.
- Ajout dans les paramètres admin un bouton `widgets`.
- Visualisation des widgets sur une page spécifique.
- Suppression du TypeEntity dans Scénario.
- Ajout de champ `cmd`.`html`.


Reste à faire :
- Stabilisation de l'affichage de la liste des widgets.
- Lors du chargement de l'édition d'un widget le select du template n'affiche pas la bonne valeur.
- Stabilisation de l'affichage de prévisualisation des widgets.
- Affectation du widget à une commande sur la page widget.
- Affectation du widget à une commande sur la page commande.
- Suppression du champs `scenario`.`type` lors d'une migration en 0.8.0
- Ajout du champs `cmd`.`html` lors d'une migration en 0.8.0
- Réalisation des tests unitaires

lucguinchard committed:
Merge pull request #1719 from lucguinchard/add-widgets

Add widgets

lucguinchard committed:
Mise à jour des tables lors d'une mise à jour 0.7.x vers 0.8.0

lucguinchard committed:
Modification de `ENGINE` en `InnoDB` de la table `interactQuery` en vu de la version 0.7.1

Esso1-jeedom commented:
fresh install 
![image](https://user-images.githubusercontent.com/40739563/77887422-d393cc80-726a-11ea-964c-6d203b0c2ae0.png)
et aprés une restoration 
![image](https://user-images.githubusercontent.com/40739563/77888370-700a9e80-726c-11ea-9388-4709df53e373.png)

lucguinchard committed:
Merge importJustAfterPostIns into add-widgets

lucguinchard committed:
Merge commit '96b64c3f4d9d075082456792879ed355f94b68c6' into add-widgets

lucguinchard committed:
Merge commit 'd31be61b0dd565e8bb4a067c1352f54a57815dd0' into add-widgets

lucguinchard committed:
Merge origin/develop into changeInteractQueryToInnoDB

lucguinchard committed:
Bugs fix lors des migrations
 - execute les fonction migate_x_y_z lors de migration nextdom
 - n'exécute pas le méthode migate_x_y_z concernant la version restauré.

lucguinchard commented:
fix https://github.com/NextDom/nextdom-core/issues/1726

slobberbone committed:
Fix migration process

All migration process will be done in 0.7.1

lucguinchard committed:
Mise en place de l'interface `widgets` pour jeedom.

lucguinchard committed:
bugfix : modification jeedom → nextdom

lucguinchard committed:
Mise ne place de la modal WidgetReplace.

lucguinchard committed:
clean scripts.cron-class-or-function-not-found

lucguinchard committed:
fix bug : les icons pur jeedom replace `nextdom` to `jeedom` - Pour la compatibilité.

Esso1-jeedom commented:
c good après la restauration d'un backup 
![image](https://user-images.githubusercontent.com/40739563/78383218-52c13180-75d8-11ea-9a45-71932afecd9a.png)

iWils reviewed (approved):
Test et approuvé:
- FirstInstall + Restore backup Nextdom 0.7.0 => Ok
- FirstInstall + Restore backup Jeedom 3.3.35 => Ok

Sylvaner committed:
Attribution du widget

lucguinchard committed:
Mise en place : possibilité de choisir une couleur pour une icone.

lucguinchard committed:
Modification Sylvain : Attribution du widget

lucguinchard committed:
Merge origin/add-widgets into add-widgets

lucguinchard committed:
nettoyage de l'affichage

lucguinchard committed:
Sélectionne les cases pour les cmds utilisées

lucguinchard committed:
change jeedom to nextdom

lucguinchard committed:
change jeedom to nextdom

lucguinchard committed:
change jeedom to nextdom

lucguinchard committed:
change jeedom to nextdom

lucguinchard committed:
change jeedom to nextdom + ajout de `;`, modification`==` en `===`...

lucguinchard committed:
bugfix $linkedEqLogic → $linkedObject

lucguinchard committed:
fix warning : quand dashboard ou mobile n'existe pas.

lucguinchard committed:
Ajout de bouton + Ajout du lien vers `La configuration de la commande`.

lucguinchard committed:
Clean

lucguinchard committed:
Mise en place des libs d'affiche displayCard

lucguinchard committed:
fix capitalizeFirstLetter

lucguinchard committed:
La génération des libs se fait dans libs et plus dans base

lucguinchard committed:
La génération des classes se fait dans base.js

lucguinchard committed:
Meuilleur gestion du panel `icon.selector`.

lucguinchard committed:
Généralisation du panel `Choisir une Icône`.

lucguinchard committed:
Généralisation du panel `Choisir une Icône` dans les Résumés domotiques.

lucguinchard committed:
Généralisation du panel `Choisir une Icône` dans les Scénarios et Objects.

lucguinchard committed:
Affiche une vignette `Vues` et une `Designs` + Ajout du nombre de tâches total si aucune n'est désactivé.

lucguinchard committed:
Ajout d'une méthode getCount à CommonManager + utilisation dans AdministrationController

lucguinchard committed:
Merge branch 'add-widgets' into changeInteractQueryToInnoDB

lucguinchard commented:
fix https://github.com/NextDom/nextdom-core/issues/1718
